src/webview/src/components/fbaCheckbox.js

```diff
@@ -202,23 +222,79 @@ export default function FBACheckbox(props) {
     );
 
     const instructionsFragment = (
         <React.Fragment>
-            <DialogContentText paragraph>
-                <TextField name='instructions' onChange={handleValueChanges} margin="dense" id={'instructions-field-' + props.name}
-                    InputLabelProps={{ shrink: true, }} fullWidth multiline value={values.instructions}
-                    label='Instructions'
-                    placeholder='Please enter instructions here on how to check whether this root cause did occur.' />
-            </DialogContentText>
+            <Divider variant="middle" />
+            <Accordion>
+                <AccordionSummary
+                    expandIcon={<ExpandMoreIcon />}
+                    aria-controls="panel1a-content"
+                    id="panel1a-header">
+                    <Typography gutterBottom variant="h5">Instructions</Typography>
+                    <IconButton size="small" aria-label="edit" onClick={() => setTestFieldEditOpen(2)} >
+                        <EditIcon fontSize="small" color="primary" />
+                    </IconButton>
+                </AccordionSummary>
+                <AccordionDetails>
+                    <TextFieldEditDialog
+                        label='Instructions'
+                        placeholder='Please enter instructions here on how to check whether this root cause did occur.'
+                        data={values.instructions || {}} onChange={(newValue) => handleValueChanges({ target: { name: 'instructions', value: newValue } })} open={textFieldExitOpen === 2} onClose={() => { setTestFieldEditOpen(0); }}
+                    />
+                    <div dangerouslySetInnerHTML={{ __html: toMarkdown(GetTextValue(values.instructions)) }} />
+                    <Divider variant="middle" />
+                </AccordionDetails>
+            </Accordion>
+        </React.Fragment>
+    );
+
+    const processingCommentsFragment = (
+        <React.Fragment>
+            <Divider variant="middle" />
+            <Accordion>
+                <AccordionSummary
+                    expandIcon={<ExpandMoreIcon />}
+                    aria-controls="panel1a-content"
+                    id="panel1a-header">
+                    <Typography gutterBottom variant="h5">Comments</Typography>
+                    <IconButton size="small" aria-label="edit" onClick={() => setTestFieldEditOpen(3)} >
+                        <EditIcon fontSize="small" color="primary" />
+                    </IconButton>
+                </AccordionSummary>
+                <AccordionDetails>
+                    <TextFieldEditDialog
+                        label='Processing comments'
+                        placeholder='Please enter some processing comments here.'
+                        data={values.comments || {}} onChange={(newValue) => handleValueChanges({ target: { name: 'comments', value: newValue } })} open={textFieldExitOpen === 3} onClose={() => { setTestFieldEditOpen(0); }}
```

Comment:
if we convert previous format .comments (strings) into objects already you can ignore the following:
value.comments is a string. Later on you assume its an object. (e.g. props.onChange({...props.data, ...}); 
This might fail if value.comments is not an object.

Reply:
I don't know yet, what we do need the "renderFormat" for. This will be persisted as well? If we don't need to persist renderFormat we might keep the previous format of .comments,... being just strings.

Reply:
renderFormat is already added for the next version. Idea is that the user can choose between, for now, markdown or raw text.

Reply:
Added option to enable / disable markdown.

> if we convert previous format .comments (strings) into objects already you can ignore the following:
> value.comments is a string. Later on you assume its an object. (e.g. props.onChange({...props.data, ...});
> This might fail if value.comments is not an object.

Due to update in *.fba format 0.3 it must always be an object now.